composer.json

```diff
@@ -17,11 +17,17 @@
         "dhii/php-cs-fixer-config": "dev-php-5.3",
         "phpunit/phpunit": "^4.8",
         "ptrofimov/xpmock": "^1.1",
-        "codeclimate/php-test-reporter": "<=0.3.2"
+        "codeclimate/php-test-reporter": "<=0.3.2",
+        "dhii/validation-interface": "^0.1"
```

Comment:
Ok, PHP 7 aside. You're clearly depending on a type here: an interface from the interface package. Even if optionally handled it is still a dependency of the package. Do you not agree?